libs/eavmlib/src/esp.erl

```diff
@@ -279,6 +281,34 @@ deep_sleep() ->
 deep_sleep(_SleepMS) ->
     erlang:nif_error(undefined).
 
+%%-----------------------------------------------------------------------------
+%% @param   Source the device that will be mounted
+%% @param   Target the path where the filesystem will be mounted
+%% @param   FS the filesystem, only fat is supported now
+%% @param   Opts
+%% @returns ok in case of success, otherwise an error
```

Comment:
Documentation of return type needs to be updated

Reply:
Fixed.